### PR TITLE
Add Season Inputs to flea api calls

### PR DESCRIPTION
**Pull request by RandalMorris**
Third time is the charm right? lol

These are the API calls that the season parameter can be added too, Without them they default to current season.

### CODE REVIEW

File: R/flea_franchises.R [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #' @export
 
 ff_franchises.flea_conn <- function(conn) {
-  x <- fleaflicker_getendpoint("FetchLeagueStandings", league_id = conn$league_id, sport = "NFL") %>%
+  x <- fleaflicker_getendpoint("FetchLeagueStandings", season = conn$season, league_id = conn$league_id, sport = "NFL") %>%
```
Comment (tanho63) [inline R/flea_franchises.R:19]: probably should have been indented like this in the first place 😅 
```suggestion
  x <- fleaflicker_getendpoint(
    "FetchLeagueStandings", 
    season = conn$season, 
    league_id = conn$league_id, 
    sport = "NFL"
  )
```

**Review by tanho63 — COMMENTED** [review]
Looks good to me! One styling suggestion (which you can implement by pressing commit-suggestion button below and then pulling the branch back down to your machine).

Can you increment the dev version in the DESCRIPTION (i.e. from 1.4.8.05 to 1.4.8.06) and add a line in NEWS.md that describes what changed?

**Comment by RandalMorris** [comment]
Added the changes.

I did notice the rosters has a scoring_period input so right now it just pulls in week 1 rosters for a given year.

File: R/flea_franchises.R (hunk 1 above)
Comment (tanho63) [reply R/flea_franchises.R]: forgot a pipe here when I made the suggestion, whoops

File: R/flea_playerscores.R [hunk 2]
```diff
@@ -25,6 +25,7 @@ ff_playerscores.flea_conn <- function(conn, page_limit = NULL, ...) {
     endpoint = "FetchPlayerListing",
     sport = "NFL",
     league_id = conn$league_id,
+    season = conn$season,
```
Comment (tanho63) [inline R/flea_playerscores.R]: seems like this parameter is actually called `sort_season` https://www.fleaflicker.com/api-docs/index.html#operation--FetchPlayerListing-get - have updated for you

File: tests/testthat/test-ff_franchises.R [hunk 3]
```diff
@@ -23,6 +23,6 @@ test_that("ff_franchises returns a tibble of franchises", {
   expect_tibble(dlf_franchises, nrows = 16)
   expect_tibble(jml_franchises, nrow = 12)
   expect_tibble(dlp_franchises, nrow = 12)
-  expect_tibble(joe_franchises, nrow = 12)
+  expect_tibble(joe_franchises, nrow = 16)
```
Comment (tanho63) [inline tests/testthat/test-ff_franchises.R:26]: This actually explains a pesky problem as to why the number of franchises had changed in this test: the league originally had 16 in 2020 but now has 12, and I couldn't figure out why this wasn't working before 😅  https://github.com/ffverse/ffscrapr/commit/1a47994d48e2159676162d8e79d2637bb7930a01

**Review by tanho63 — APPROVED** [review]
Thanks for the additions! Have made a few styling things and added the week arg to `ff_rosters()` you were mentioning. This'll merge when the tests pass 🤞

**Pull request merged**